modules/vaos/app/services/vaos/v2/appointments_reason_code_service.rb

```diff
@@ -4,6 +4,14 @@ module VAOS
   module V2
     # This class contains the logic for extracting extra fields from reason codes and updating the appointment.
     class AppointmentsReasonCodeService
+      # Reason code purpose text
+      PURPOSE_TEXT = {
```

Comment:
Note that this is using slightly different logic than the original [here](https://github.com/department-of-veterans-affairs/vets-website/blob/5fc5b4a339e34be9db43e680a4c35798b963ff9b/src/applications/vaos/services/appointment/transformers.js#L362-L366).
1. The logic here compares using the `commentShort` field listed [here](https://github.com/department-of-veterans-affairs/vets-website/blob/5fc5b4a339e34be9db43e680a4c35798b963ff9b/src/applications/vaos/utils/constants.js#L33-L62) only instead of using `serviceName` as well.
2. The logic here allows for the 'OTHER_REASON' coding